examples/README.md

```diff
@@ -0,0 +1,43 @@
+# Rapid Java SDK Examples
+
+This repository contains examples of how to use the Expedia Group Rapid Java SDK. The examples are written in Java and use Maven for dependency management.
+
+## Examples
+
+The example implementation provided in the examples package demonstrate different scenarios utilizing various RAPID APIs through the SDK:
+
+The scenarios below specifically demonstrate the usage of booking APIs:
+
+1.`SingleRoomBookScenario.java`: This example demonstrates the process of booking a single room. It includes steps like getting property availability, checking room prices, and finally booking a room.
+
+2.`MultiRoomHoldAndResumeBookScenario.java`: This example demonstrates how to hold multiple rooms for booking and then resume the booking process. It includes steps like creating a booking with hold, resuming the booking process, and verifying the booking has been resumed properly.
+
+More scenarios are included in the examples package to demonstrate the usage of other RAPID APIs.
+
+## Setup
+
+1. Clone the repository.
+2. Navigate to the project directory `examples`.
+3. Acquire your own API key and secret from Expedia Group. Replace the placeholders `your_api_key` and `your_api_secret` in the `RapidService` class with your actual API key and secret.
```

Comment:
This should be a prerequisite, we can move it above, or rewrite it to something like: `Make sure you already have your own API key....`

We could also add a link to the [Getting started with Rapid](https://developers.expediagroup.com/docs/products/rapid/setup/getting-started#getting-started-with-rapid) page.